### PR TITLE
Pia 4103 update customer documentation for customization of background colors on all help screens

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**
PIA-4103
- update image assets
- add extra text for screen background color and cell background(if needed) in documentation for all help screens
- change order for help screens

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md [hunk 1]
```diff
@@ -320,3 +282,104 @@ Some background and text colors use the `GiniColor` type with which you can set
...
+	  - Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.help.backToMenu*</span> localized string
+  - With title only
+	  - Title &#8594; `GiniConfiguration.navigationBarHelpScreenTitleBackToMenuButton`
+
+##### 1. Header
+
+- Text &#8594; <span style="color:#009EDF">*ginicapture.help.openWithTutorial.collectionHeader*</span> localized string
+
+##### 2. Open with steps
+
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md:367]: [Suggestion] @ValentinaIancu-Gini could you please align the formatting in the sections for help screens? 🙏

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great job 🥳,
please ignore my separate formatting suggestions (like remove the line).
Could you please align the formatting for the all help screens? (like remove a line after the enumeration)?
I've downloaded the documentation and notices that for Help screen we have 1..4 options, but on the screenshot we have 3, could you please update it?
Screenshot 2023-05-24 at 18 16 33

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md (hunk 1 above)
Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md]: done

File: CaptureSDK/GiniCaptureSDK/Documentation/jazzy-theme/assets/img/Customization guide/No results.png
[the remark this answers is not in the document]
Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Documentation/jazzy-theme/assets/img/Customization guide/No results.png:1]: done

File: CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md [hunk 2]
```diff
@@ -320,3 +282,104 @@ Some background and text colors use the `GiniColor` type with which you can set
...
+Overriding tips images below will lead to the changes on the [Capturing tips screen](#capturing-tips-screen).
+
+- Tip 1 image &#8594; <span style="color:#009EDF">*captureSuggestion1*</span> image asset
+- Tip 2 image &#8594; <span style="color:#009EDF">*captureSuggestion2*</span> image asset
+- Tip 3 image &#8594; <span style="color:#009EDF">*captureSuggestion3*</span> image asset
+- Tip 4 image &#8594; <span style="color:#009EDF">*captureSuggestion4*</span> image asset
+- Tip 5 image &#8594; <span style="color:#009EDF">*captureSuggestion5*</span> image asset
+
+##### 3. Background
+
```
[the remark this answers is not in the document]
Comment (ValentinaIancu-Gini) [reply CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md]: done

**Review by zladzeyka — APPROVED** [review]
👏🏻

**Pull request merged**